spacy/displacy/__init__.py

```diff
@@ -184,8 +184,17 @@ def parse_ents(doc: Doc, options: Dict[str, Any] = {}) -> Dict[str, Any]:
     doc (Doc): Document do parse.
```

Comment:
Entirely unrelated to your PR, but it bugs me ;-)
```suggestion
    doc (Doc): Document to parse.
    options (Dict[str, Any]): NER-specific visualisation options.
```